src/templates/module_test.html

```diff
@@ -0,0 +1,28 @@
+<!-- Bootstrap HTML for running the unit tests. -->
+<!DOCTYPE html>
+<html>
+
+<head>
+  <script type="text/javascript">
+    window.logs = [];
+    console.log = function (message) {
+      window.logs.push(message);
+    };
+    console.warn = function (message) {
+      window.logs.push(message);
+    };
+    console.info = function (message) {
+      window.logs.push(message);
+    };
+    console.error = function (message) {
+      window.logs.push(message);
+    };
+  </script>
+  <script type="module">
+    import { loadPyodide } from './pyodide.mjs'
```

Comment:
Won't you need to add this to the global scope? Otherwise it won't be possible to run `loadPyodide` from `selenium.run_js`.

Reply:
I think when we `import`, `loadPyodide` is automatically added to `global`.  But let's see how the tests run.

Reply:
I think the `<script type="module">` comes with it's own scope and everything in `<script type="module">` is limited to this scope unless otherwise stated.

Reply:
Ahh I think you're correct. Hmm ~ Let me fix that